src/newSeed/stage8.html

```diff
@@ -19,7 +19,7 @@
           ${wizardState.seedTokenSymbol} to the Seed contract.
           This amount includes any service fees.
         </p>
-        <p><etherscanlink address.to-view="wizardState.seedAdminAddress" text.to-view="wizardState.seedAdminAddress | smallHexString"></etherscanlink> is the sole administrator of the new Seed<span if.to-view="!!seedConfig.seedDetails.whitelist"> and are responsible for managing its whitelist</span>.</p>
+        <p><etherscanlink address.to-view="wizardState.seedAdminAddress" text.to-view="wizardState.seedAdminAddress | smallHexString"></etherscanlink> is the sole administrator of the new Seed<span if.to-view="!!wizardState.whiteList"> and <a href="${wizardState.whiteList}" target="_blank" rel="noopener noreferrer">${wizardState.whiteList} </a> are responsible for managing its whitelist</span>.</p>
```

Comment:
why the extra "`<a href="${wizardState.whiteList}" target="_blank" rel="noopener noreferrer">${wizardState.whiteList} </a>`"?  Have you tested this?

Reply:
I thought the whitelist address was to be displayed

Reply:
Unless someone is asking for it, then please don't add it.  If they are asking for it, then we need to wording of the sentence to make sense.

In any case, "and are responsible" should be "and is responsible".